apps/token/src/routes/governance/components/proposal-detail-header/proposal-header.spec.tsx

```diff
@@ -0,0 +1,243 @@
+import { render, screen } from '@testing-library/react';
+import { generateProposal } from '../../test-helpers/generate-proposals';
+import { ProposalHeader } from './proposal-header';
+import type { Proposals_proposals } from '../../proposals/__generated__/Proposals';
+
+const proposal = generateProposal();
+
+const renderComponent = (proposal: Proposals_proposals) => (
+  <ProposalHeader proposal={proposal} />
+);
+
+it('New market', () => {
+  render(
+    renderComponent({
+      ...proposal,
```

Comment:
chore (blocking): For all these tests it would be better to generateProposal for each test. Then you are safer from mutating the proposal between tests and you also can remove the proposal and proposal.terms spread